net-wireless/wepattack/wepattack-0.1.3-r5.ebuild

```diff
@@ -22,14 +23,21 @@ DEPEND="
 "
 
 RDEPEND="${DEPEND}
-john? ( || ( app-crypt/johntheripper app-crypt/johntheripper-jumbo ) )"
+	john? ( || (
+		app-crypt/johntheripper
+		app-crypt/johntheripper-jumbo
+	)
+)"
```

Comment:
one more indentation for the last two lines

Reply:
done